test/testapps/AppWithP2PReference/AppWithP2PReference.csproj

```diff
@@ -1,10 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk.Web" />
```

Comment:
@pranavkm I ran into some problems when I started messing with the import order. It got a lot more complicated once I was no longer using Directory.props to import things.

However what's here now basically will reflect the final state once we're integrated with the CLI.